pkgs/development/python-modules/heudiconv/default.nix

```diff
@@ -23,9 +25,19 @@ buildPythonPackage rec {
 
   src = fetchPypi {
     inherit pname version;
-    sha256 = "sha256-UUBRC6RToj4XVbJnxG+EKdue4NVpTAW31RNm9ieF1lU=";
+    hash = "sha256-UUBRC6RToj4XVbJnxG+EKdue4NVpTAW31RNm9ieF1lU=";
   };
 
+  postPatch = ''
+    # the package builds without versioningit
+    sed -i '/versioningit ~=/d' pyproject.toml
```

Comment:
nitpick but this should use substituteInPlace

Reply:
I wanted to match only the part before the version number, and I wasn't sure substituteInPlace could do that (or delete a line).

Reply:
Ah I see, tbh I don't know and guess this is fine then.